src/server/game/AuctionHouseBot/AuctionHouseBot.cpp

```diff
@@ -146,7 +146,6 @@ void AuctionBotConfig::GetConfigFromFile()
     SetConfigMax(CONFIG_AHBOT_NEUTRAL_ITEM_AMOUNT_RATIO, "AuctionHouseBot.Neutral.Items.Amount.Ratio", 100, 10000);
 
     SetAHBotIncludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceIncludeItems", ""));
-    SetAHBotExcludes(sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", ""));
```

Comment:
I only used std::string as an example but what about

```C++
std::string disabledItems = sConfigMgr->GetStringDefault("AuctionHouseBot.forceExcludeItems", "")
if (boost::lexical_cast<bool>(disabledItems))
{
    // do stuff
}

As far as losing data on a db update, you would lose that anyway because those entries are entirely user defined so the user would need to maintain a separate SQL file for their AHBot stuff, along with the other AHBot items in the CONF file.